### DISCUSSION TITLE
CryptoSmite explanation doesn't mention if the kernver ends with 4

### DISCUSSION THREAD

**Discussion by Bee4Queer** [3kh0/ext-remover#1289] [Fixes JOIN TITANIUM NETWORK TO SHARE PLEASE]
I'm guessing that the thing for 3 also applies to 4, but it isn't mentioned.

**Comment by schoolexploitkid** [comment]
yes, it applies for 4, lemme make pr

**Comment by Charlieis1nsan3** [comment]
because kernver 4 isn't fucking available. and check your version if your anything above v120 it doesn't fucking work -_-

**Reply by Charlieis1nsan3** [reply]
like I'm not trying to be a dick. but I think that's pretty obvious

**Reply by Bee4Queer** [reply]
Yeah, it is obvious to me and you, but have you _seen_ the details for LTMEAT? Like half of the explanation is just for idiots/skids! So sure, it _seems_ obvious, but then you're going to have someone asking some dumb question because they assumed that kernver 4 worked, bricked their chromebook, and now they are asking how they can unbrick it before school in half an hour.

**Reply by Bee4Queer** [reply]
Also, when I get a chance I'll send you a picture of my kernver(iirc 0x0(dunno how many zeros)01004)

**Reply by Charlieis1nsan3** [reply]
k

**Reply by Bee4Queer** [reply]
![image](https://github.com/user-attachments/assets/200c0603-1b26-416e-86ce-f0fbcefcb97a)

**Reply by Charlieis1nsan3** [reply]
what version are you?

**Reply by Charlieis1nsan3** [reply]
you might be able to do KVS

**Reply by Bee4Queer** [reply]
128.0.6613.161

**Reply by Bee4Queer** [reply]
Also, on rigtools connected to goguardian, it mentions "Riienrollment". Can you shed any light on what that is?

**Reply by Charlieis1nsan3** [reply]
I personally never used rigtools because I didn't find out until later but you can possible disable extensions using rigtools if you have access to Chrome://extensions

**Reply by Charlieis1nsan3** [reply]
like Exthang3r

**Comment by Bee4Queer** [comment]
Anyway, #1290 fixes this. I'll wait to close until it gets merged.

**Comment by Bee4Queer** [comment]
#1290 has been merged.
